docs/book/src/topics/secondary-load-balancer.md

```diff
@@ -34,3 +34,32 @@ spec:
     name: internal-apiserver
     scheme: internal     # optional
 ```
+
+## Target Group IP Address Type
+
+**Note:** The `targetGroupIPType` field is only available when using Network Load Balancers (NLB), Application Load Balancers (ALB), or Gateway Load Balancers (ELB). It **cannot** be configured when using Classic Load Balancers.
```

Comment:
Secondary LB should never be a classic load balancer, iirc.

Reply:
Thanks, addressed!